tests/main_test.go

```diff
@@ -39,11 +38,11 @@ var (
 )
 
 const (
-	tNodeIdx1      = 0
-	tNodeIdx2      = 1
-	tNodeIdx3      = 2
-	tNodeIdx4      = 3
-	tTotalNodes    = 4 // each node has 3 validators
+	tNodeIdx1 = iota
```

Comment:
It is not an enumerator. Please revert this.

Reply:
Fixed.